plugins/package-managers/swiftpm/src/main/kotlin/SwiftPmModel.kt

```diff
@@ -52,6 +52,26 @@ data class SwiftPackage(
     )
 }
 
+/**
+ * See https://github.com/apple/swift-package-manager/blob/3ef830dddff459e569d6e49c186c3ded33c39bcc/Sources/PackageGraph/PinsStore.swift#L387-L462.
+ */
+@Serializable
+data class PinV2(
```

Comment:
Not introduced in this commit, but shouldn't these classes be internal instead of public?

Reply:
I don't mind if it's public or internal, what do you prefer?
(But I believe the visibility should be consistent with the two `parse*()` functions below.

Reply:
Done in commit on top.